cosmosdb/Ballerina.toml

```diff
@@ -2,7 +2,7 @@
 distribution = "2201.0.0"
 org= "ballerinax"
 name= "azure_cosmosdb"
-version= "2.2.0"
+version= "3.0.0"
```

Comment:
This version change will happen in Depenedencies.toml also.
Please commit that change also

Reply:
Updated